data-sample/cmaq/form-ald2-aldx/form-ald2-aldx-update-table.sql

```diff
@@ -0,0 +1,39 @@
+-- Format
+DROP TABLE IF EXISTS form_ald2_aldx;
+
+-- create temp table to hold new data
+CREATE TEMP TABLE IF NOT EXISTS form_ald2_aldx (
```

Comment:
```IF NOT EXISTS``` not needed since the table ```form_ald2_aldx``` was just dropped, so it certainly does not exist, right?

Reply:
These scripts can also be copied/pasted out of context of the development environment, so having it there may make sense if not running this via a loader.

Reply:
@mjstealey OK. That justifies it.